tools/downloader/pytorch_to_onnx.py

```diff
@@ -58,6 +58,8 @@ def parse_args():
                         help='Space separated names of the output layers')
     parser.add_argument('--model-param', type=model_parameter, default=[], action='append',
                         help='Pair "name"="value" of model constructor parameter')
+    parser.add_argument('--onnx-version', type=int, default=9,
+                        help='Version of symbolic opset')
```

Comment:
You should set default to `11`  (see, [PR](https://github.com/openvinotoolkit/open_model_zoo/pull/1638) for details.
cc @IRDonch